cmd/juju/charmhub/download.go

```diff
@@ -102,6 +104,7 @@ func (c *downloadCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.IntVar(&c.revision, "revision", -1, "specify a revision of the charm to download")
 	f.StringVar(&c.archivePath, "filepath", "", "filepath location of the charm to download to")
 	f.BoolVar(&c.noProgress, "no-progress", false, "disable the progress bar")
+	f.BoolVar(&c.resources, "resources", false, "download the resources associated with the charm")
```

Comment:
> Need to handle downloading resources by revision.

We probably should have resource downloading as a separate command. This specifically is resolve the charm and download it, optionally with the resources.

> Should we download the resources for a charm which has them rather than requiring op in?

Yeah I'd be happy with a `--no-resources` argument instead.